extensions/standard-processors/processors/GetFile.cpp

```diff
@@ -232,27 +232,24 @@ std::queue<std::string> GetFile::pollListing(uint64_t batch_size) {
 bool GetFile::fileMatchesRequestCriteria(std::string fullName, std::string name, const GetFileRequest &request) {
   logger_->log_trace("Checking file: %s", fullName);
 
-#ifdef WIN32
-  struct _stat64 statbuf;
-  if (_stat64(fullName.c_str(), &statbuf) != 0) {
-    return false;
+  std::error_code ec;
+  uint64_t file_size = std::filesystem::file_size(fullName, ec);
+  if (ec) {
+    logger_->log_error("file_size of %s: %s", fullName, ec.message());
```

Comment:
I think a `return false;` is missing after this line

Reply:
Added in [2889dd4](https://github.com/apache/nifi-minifi-cpp/pull/1311/commits/2889dd4ae96f7bcad1d5fdf552ad22198729991c). Sorry for the long turnaround time.